plugins.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Plugins
+headline: Pico Plugins
+description: |
+  Want to extend the functionality of Pico?  Below is a collection of our community-developed plugins.<br>
+  Would you like to create your own plugin? Learn how in [our documentation](/docs/#plugins)!
+nav: 5
+
+portfolio:
+  layout: card
+  categories:
+    admin: Administration
+    developer: Developer-Focused
+    theming: Theming Extensions
+  defaultThumbnail: plugins/images/thumbnails/plugin.png
+  defaultImage: themes/images/fullsize/default_theme.png
+
+---
+
+{% include portfolio.html portfolio=site.plugins %}
+
+---
+
+## More Plugins & How to Contribute Your Own
+
+The plugins shown here are only a smaller selection of those available.  You can find more themes on [our Wiki][].
+
+You can even [contribute][] your own plugin to the community and we'll feature it here.  While you can freely add your plugin to [our Wiki][], if you'd also like to be included on this page, you can follow the simple instructions in our [Submission Guidelines][].
```

Comment:
I *really* like the idea of a big plus tile. `[ + ]` :+1: :+1: :+1: 

I don't think it's a good place to put the *entire* submission guidelines though.

I *would* have said that the content area likes to close itself too often to have long content, but you fixed that with your "Rage Refactoring". :laughing: :wink:

I do think it would be a little hard to read though.  Especially with white text on black. (And changing it to somehow have a white background inside the content area sounds odd).

I'm thinking maybe make it a regular item, with a special fullsize image and description pointing to the regular Submission Guidelines page.

For the fullsize image, maybe something like a collage of themes and code or something split diagonally between a theme and its code `[T/C]`.

Finally, I *don't* like the idea of removing the existing "Contributing" section from the page.  Not sure if you meant that this "boring" text should be removed or not, but my vote is no.

There's no reason we couldn't have it in both places.  :wink:

I just don't want it to be harder to find.  It should state clearly on the page that we want contributers.

Also... the "Wiki" parts of that section aren't big enough to stand on their own. :laughing: